horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/ActionServiceImpl.java

```diff
@@ -160,27 +161,52 @@ void validate(Action action) {
     @WithRoles
     @Transactional
     @Override
-    public Action add(Action action) {
+    public Action addGlobalAction(Action action) {
```

Comment:
Actually, do we want to restrict this to Admin role?

Reply:
It is already, that restriction is truncated in the diff changes 😄 but if you extend it on top you would see it
![image](https://github.com/user-attachments/assets/ddbd812c-7634-416c-8da2-940f50f2f0c6)